examples/ActionGroup.Tests.ps1

```diff
@@ -16,6 +16,23 @@ Describe 'Verify Action Group Exists' {
   }
 }
 
+Describe 'Verify Action Group Does Not Exist' {
+  it 'Should not contain a action group named sampleActionGroup' {
+    #arrange
+    $resourceGroupName = "test-rg"
+    $actionGroupName = "sampleActionGroup"
+
+    #act
+    # The '-ErrorAction SilentlyContinue' command
+    # suppresses errors when the underlying functions
+    # don't find the resource; remove this to see the error
```

Comment:
Nit: -ErrorAction SilentlyContinue will suppress all errors. Perhaps edit these comments to more accurately reflect that? Something like `The '-ErrorAction SilentlyContinue' command suppresses all errors. In this test it will suppress the error message when a resource cannot be found...`